.github/workflows/build_flang_arm64.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Build and install flang & libpgmath
         run: |
           cd ${{ env.build_path }}/flang
-          ./build-flang.sh -t ${{ matrix.target }} -p ${{ env.install_prefix }} -n `nproc --ignore=1` -v
+          ./build-flang.sh -t ${{ matrix.target }} -p ${{ env.install_prefix }} -n `nproc --ignore=1` -v -l /home/root/classic-flang-llvm-project/llvm
```

Comment:
`/home/root`?

Reply:
Yes, `/home/root` is the location of the latest `classic-flang-llvm-project` source tree for the AArch64 build job. [It is hardcoded in the Docker image.](https://github.com/flang-compiler/classic-flang-llvm-project/blob/release_15x/.github/workflows/Ubuntu20Dockerfile#L57)